core/modules/layout/layout.module

```diff
@@ -1072,7 +1072,7 @@ function layout_get_layout_info($layout_name = NULL) {
       $info[$name] += array(
         'preview' => 'preview.png',
         'stylesheets' => array(
-          'all' => array('layout.css'),
+          'all' => array('one-column.css'),
```

Comment:
This would make it so that `one-column.css` would be used for all Layouts. Could you make it be a str_replace() to use the layout name instead? See the lines just below that do this for the layout .tpl.php file.